src/opnsense/mvc/app/models/OPNsense/TrafficShaper/TrafficShaper.xml

```diff
@@ -63,11 +63,21 @@
                       <rr>Deficit Round Robin</rr>
                       <qfq>QFQ</qfq>
                       <fq_codel>FlowQueue-CoDel</fq_codel>
+                      <fq_pie>FlowQueue-PIE</fq_pie>
                   </OptionValues>
                 </scheduler>
                 <codel_enable type="BooleanField">
                   <default>0</default>
                   <Required>Y</Required>
+                  <Constraints>
+                      <check001>
+                          <ValidationMessage>Only one option could be selected</ValidationMessage>
```

Comment:
@mimugmail maybe we should be a little more explicit about which options interfere, something like `"FlowQueue-CoDel and FlowQueue-PIE can not be combined"`?

Reply:
Good point, I'll also fix some indents later today

Reply:
thanks!